src/utils/convert-filter.ts

```diff
@@ -1,6 +1,11 @@
 import escape from 'escape-regexp'
 import mongoose from 'mongoose'
 
+const FIND_ONE_FIELDS = [
+  '_id',
+  'uuid',
```

Comment:
While `_id` is pretty standard, `uuid` less so. I think we should make this configurable somehow, which also lets us change the Admin app without having to change this library every time we want to add a new field to this.